### PR TITLE
Added scrutinizerci scanner

### PR DESCRIPTION
**Pull request by roxanne-tampus**

### CODE REVIEW

File: pkg/detectors/scrutinizerci/scrutinizerci.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"scrutinizerci"}) + `\b([0-9a-z]{64})\b`)
```
Comment (dustin-decker) [inline pkg/detectors/scrutinizerci/scrutinizerci.go:23]: Please change this to be just `scrutinizer` to cache `scrutinizer ci`, scrutinizer-ci`, `scrutinizer`, etc

Comment (dustin-decker) [reply pkg/detectors/scrutinizerci/scrutinizerci.go:23]: Same with L29 too

Comment (roxanne-tampus) [reply pkg/detectors/scrutinizerci/scrutinizerci.go]: Commited the changes

File: pkg/detectors/scrutinizerci/scrutinizerci.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"scrutinizer"}) + `\b([0-9a-z]{64})\b`)
+)
+
+// Keywords are used for efficiently pre-filtering chunks.
+// Use identifiers in the secret preferably, or the provider name.
+func (s Scanner) Keywords() []string {
+	return []string{"scrutinizerci"}
```
[the remark this answers is not in the document]
Comment (roxanne-tampus) [reply pkg/detectors/scrutinizerci/scrutinizerci.go]: Done

**Pull request merged**